modules/ROOT/pages/index.adoc

```diff
@@ -1,77 +1,82 @@
 [[index]]
 = Introduction
 :page-aliases: introduction.adoc
-:description: This section describes the Neo4j GraphQL Library.
+:description: An introduction to the Neo4j GraphQL Library.
 
-The Neo4j GraphQL Library is a highly flexible, low-code, open source JavaScript library that enables rapid API development for cross-platform and mobile applications by tapping into the power of connected data.
+The Neo4j GraphQL Library is:
 
-With Neo4j as the graph database, the GraphQL Library makes it simple for applications to have data treated as a graph natively from the frontend all the way to storage.
-This avoids duplicate schema work and ensures flawless integration between frontend and backend developers.
-
-If you are new to Neo4j and GraphQL take a look at xref:getting-started/index.adoc[Creating a new project] and xref:getting-started/toolbox.adoc[Neo4j GraphQL Toolbox] to learn the fundamentals of the Neo4j GraphQL Library and how to create GraphQL APIs backed by a Neo4j graph database.
+* highly flexible and low-code
+* open source
+* a JavaScript library
+* leveraging graph-native data
```

Comment:
I'm not 100% sure why this was moved to be a list, it feels much more concise to keep it as a single sentence, and maybe use the space for a reduced list of features to showcase at the beginning?